File: tp2/punto2/Cancion.py
class Cancion:
    def __init__(self):
        self.__nombre = None
        self.__autor = None
        self.__duracion = 0.0

    @property
    def nombre(self):
        return self.__nombre
    
    @nombre.setter
    def nombre(self, nombre):
        self.__nombre = nombre
    
    @property
    def autor(self):
        return self.__autor
    
    @autor.setter
    def autor(self, autor):
        self.__autor = autor

    @property
    def duracion(self):
        return self.__duracion
    
    @duracion.setter
    def duracion(self, duracion):
        self.__duracion = duracion

File: tp2/punto2/Cancion.java
public class Cancion {
    private String nombre;
    private String autor;
    private int duracion;

    public Cancion(String nombre, String autor, int duracion) {
        this.nombre = nombre;
        this.autor = autor;
        this.duracion = duracion;
    }

    public Cancion(){
        this.nombre = "";
        this.autor = "";
        this.duracion = 0.0;
    }

    public String getNombre() {
        return nombre;
    }

    public String getAutor() {
        return autor;
    }

    public int getDuracion() {
        return duracion;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    public void setAutor(String autor) {
        this.autor = autor;
    }

    public void setDuracion(int duracion) {
        this.duracion = duracion;
    }

    public String toString() {
        return "Nombre: " + nombre + "\nAutor: " + autor + "\nDuracion: " + duracion;
    }
}